build-and-stage-preconfigured.ps1

```diff
@@ -26,15 +26,17 @@ $selectedSection = if ($IsOnWindowsOS) { "win" } else { "mac" }
 $packageAndFeatures = $packageInfo.$selectedSection.package
 $linkType = $packageInfo.$selectedSection.linkType
 $buildType = $packageInfo.$selectedSection.buildType
+$vcpkgHash = $packageInfo.$selectedSection.vcpkgHash
 
 Write-Host ""
 Write-Host "Variables set based on config for OS `"$selectedSection`":"
 $allParams = @{
     PackageDisplayName = $PackageDisplayName
     packageAndFeatures = $packageAndFeatures
-    linkType = $linkType
-    buildType = $buildType
-    tagBaseName = $tagBaseName
+    LinkType = $linkType
+    BuildType = $buildType
+    ReleaseTagBaseName = $tagBaseName
```

Comment:
It took me a while to figure out what "tag" was, so the naming of `ReleaseTagBaseName` is an improvement.

That makes me wonder why we need the "tag" to be different than the "name". It looks like the only case it matters is `openssl (static)` vs `openssl-static`. What if we just renamed it to `openssl` and remove the need to deal with "tags" at all?

Reply:
My thought was that, if we use this beyond just in Camtasia or even CommonCpp, we might be using a statically linked version of a lib sometimes and using it dynamically linked elsewhere.  I think this is true of OpenSSL currently, but I'll have to double check that, though.

Reply:
Another option would be to by default use the base library name as the tag, and allow us to just optionally override what the tag base name will be in a preconfigured package (if it exists in the json).

Reply:
Can we not call it a "tag"? That means too many things to me, and it's too git-specific. Maybe "publishName"?

Reply:
Well, we could just get rid of "tag" altogether, and have name and tag just be the same thing.  I could just rename what shows up in that dropdownlist in Azure Devops to just say "openssl-static".  It might be nice as a user to know what you pick there directly corresponds to what the release name will end up being.

Reply:
Yeah I like the idea of getting rid of "tag" even better 😄